.eslintrc.js

```diff
@@ -4,50 +4,10 @@ module.exports = {
     ecmaVersion: 2017,
     sourceType: 'module'
   },
-  plugins: ['ember', 'prettier'],
-  extends: [
-    'eslint:recommended',
-    'plugin:ember/recommended',
-    'plugin:prettier/recommended'
-  ],
+  plugins: ['prettier'],
```

Comment:
Can you rebase this PR from `master` rather than the merging commits? These files don't seem like they should be changed by your PR.